src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -8046,15 +8046,16 @@ def test_gallery_application(self, resource_group, resource_group_location):
         self.cmd('sig create -r {gallery} -g {rg}')
         self.cmd('sig gallery-application create -n {app_name} -r {gallery} --os-type windows -g {rg}', checks=[
             self.check('name', '{app_name}'),
-            self.check('supportedOsType', 'Windows'),
+            self.check('supportedOSType', 'Windows'),
             self.check('description', None),
             self.check('tags', None),
             self.check('type', 'Microsoft.Compute/galleries/applications')
         ])
+        self.cmd('sig gallery-application wait -n {app_name} -r {gallery} -g {rg} --exists')
         self.cmd('sig create -r {gallery} -g {rg}')
         self.cmd('sig gallery-application update -n {app_name} -r {gallery} -g {rg} --description test --tags tag=test', checks=[
             self.check('name', '{app_name}'),
-            self.check('supportedOsType', 'Windows'),
+            self.check('supportedOSType', 'Windows'),
```

Comment:
since the output field has been changed and it is actually a breaking change, it is better to add a changelog to the History Notes section like this:
![image](https://github.com/user-attachments/assets/9492f0b1-8da8-43d4-9b7f-e4a6cc60ea97)

Reply:
updated comment